packages/babel-runtime-corejs2/regenerator/index.js

```diff
@@ -1 +1,15 @@
-module.exports = require("../helpers/regeneratorRuntime")();
+// TODO(Babel 8): Remove this file.
+
+const runtime = require("../helpers/regeneratorRuntime")();
+module.exports = runtime;
+
+// Copied from https://github.com/facebook/regenerator/blob/main/packages/runtime/runtime.js#L736=
+try {
+  regeneratorRuntime = runtime;
+} catch (accidentalStrictMode) {
+  if (typeof globalThis === "object") {
+    globalThis.regeneratorRuntime = runtime;
```

Comment:
Can we remove the global definition behind the `BABEL_8_BREAKING` flag?

Reply:
No, this file isn't compiled.